cfgov/agreements/jinja2/agreements/index.html

```diff
@@ -49,9 +49,20 @@ <h2 id="issuer-select">
                     for agreement submission instructions.
                 </p>
                 <ul class="m-list m-list__links u-mt15 cc-links">
+                    <li class="m-list m-list__links">
+                      <span>
+                          Q1 agreements may include omissions due to the Bureau’s COVID-19
+                          <a href=" https://files.consumerfinance.gov/f/documents/cfpb_data-collection-statement_covid-19_2020-03.pdf">
+                              regulatory flexibility statement
+                          </a>.
+                      </span><br/>
```

Comment:
```suggestion
                    <li class="m-list m-list__links">
                      <span>
                          Q1-2020 agreements may include omissions due to the Bureau’s COVID-19
                          <a href=" https://files.consumerfinance.gov/f/documents/cfpb_data-collection-statement_covid-19_2020-03.pdf">
                              regulatory flexibility statement
                          </a>.
                      </span><br/>
                    </li>
                    <li class="m-list m-list__links">
                      <span>
```

I think wrapping the note about Q1-2020 in its own li+span honors the current layout of the page a little better. Also it's probably worth changing Q1 to Q1-2020 in the note.
<img width="658" alt="Screen Shot 2020-06-05 at 12 32 33 PM" src="https://user-images.githubusercontent.com/1558033/83915909-081f7b00-a729-11ea-9a27-cbe7b3ad08ae.png">

Reply:
The Q2-2019 agreements also has a disclaimer, and it doesn't use a separate `li` for its disclaimer. Do you think we should do both disclaimers the same way, or should this one be different because it's at the top of the list?

Reply:
My thinking was that it looked too crowded with the link to the statement immediately above the download link & I guessed that this might apply for Q2 as well, so we probably don't want to repeat it & instead give it its own `li`. Probably not really an issue either way & this page is much more about the content than stellar design.

Reply:
Wait, are you saying I should I change Q2-2019 as well, or no?

Reply:
Sorry, I meant the note added in this PR might later apply to Q2-2020 as well, since the text of the data collection statement includes "until further notice", meaning we'd probably keep it at the top and change the text to "Q1-2020 and Q2-2020..."

Reply:
Ah, got it. Sounds good to me. 👍